search/src/components/RegisterOrUserProfile.tsx

```diff
@@ -54,22 +56,28 @@ const RegisterOrUserProfile = () => {
     });
   });
 
+      createEffect(async()=>{
+      const response = await fetch(`https://api.github.com/repos/arguflow/arguflow`);
```

Comment:
You'd want to fetch this on the server side at search/src/layouts/SearchLayout.astro so that's called only once. Also, @skeptrunedev we may also add a GITHUB_API_KEY to be able to perform more requests if available as GitHub API has very hard limits.

Reply:
Ah, that's interesting. Good to know. Will make those changes before merging this.